NOTICES.txt

```diff
@@ -1,586 +1,724 @@
-
-THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
-
---------------- APPENDIX 3: MIT License (Template) -----------
-
-Copyright <YEAR> <COPYRIGHT HOLDER>
-
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+=============== TABLE OF CONTENTS =============================
+
+The following is a listing of the Secrets Provider for K8s open source components detailed
+in this document. This list is provided for your convenience; please read
+further if you wish to review the copyright notice(s) and the full text
+of the license associated with each component.
+
+
+Section 1: Apache-2.0
+>>> github.com/cyberark/conjur-api-go - v0.6.0
+>>> github.com/cyberark/conjur-authn-k8s-client - v0.16.1
+>>> github.com/googleapis/gnostic - v0.3.1
+>>> github.com/modern-go/reflect2 - v1.0.1
+>>> k8s.io/api - v0.0.0-20190313235455-40a48860b5ab
+>>> k8s.io/apimachinery - v0.0.0-20190313205120-d7deff9243b1
+>>> k8s.io/client-go - v11.0.0
+>>> k8s.io/klog - v1.0.0
+>>> gopkg.in/yaml.v2 - v2.2.4
+>>> k8s.io/utils - v0.0.0-20191218082557-f07c713de883
+
+Section 2: MIT
+>>> github.com/cenkalti/backoff - v2.2.1
+>>> github.com/json-iterator/go - v1.1.9
+>>> github.com/smartystreets/goconvey - v0.0.0-20190731233626-505e41936337
+>>> sigs.k8s.io/yaml - v1.1.0
+
+Section 3: BSD 3-Clause
```

Comment:
"New" or "Revised" License was removed? Is it not a new or revised license anymore?

Reply:
it's the same license - see https://opensource.org/licenses/BSD-3-Clause
> The 3-Clause BSD License
> SPDX short identifier: BSD-3-Clause
> Note: This license has also been called the "New BSD License" or "Modified BSD License".